csharp_dotnetcore/14.NLP-With-Dispatch/NLP-With-Dispatch-Bot.bot

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "NLP-With-Dispatch-Bot",
+    "description": "",
+    "secretKey": "",
+    "services": [
+        {
+            "type": "endpoint",
+            "name": "nlp-with-dispatch",
+            "id": "http://localhost:3978/api/messages",
+            "appId": "",
+            "appPassword": "",
+            "endpoint": "http://localhost:3978/api/messages"
+        },
+        {
```

Comment:
See my other comment about app insights below. The bot file should not have references to this.